src/insns/cbld_32bit.adoc

```diff
@@ -25,6 +25,7 @@ Copy `cs2` to `cd` and set `cd.tag` to 1 if
 
 . `cs1.tag` is set, and
 . `cs1` 's bounds are not <<section_cap_malformed,malformed>>, and all reserved fields are zero, and
+. `cs1` 's permissions could have been legally produced by <<ACPERM>>, and
```

Comment:
Both of these are implied by the tag being set

Reply:
That's true, but I think it still makes sense to write it here explicitly.

Reply:
they are - but we're including cases where the memory system has faults in and so can't be relied upon just like in https://github.com/riscv/riscv-cheri/pull/285

so if someone does a rowhammer style attack for example, resulting in a cap which is obviously faulty, then detect it when possible and clear the tag

if the implementer isn't interested in such cases then they don't have to implement the logic

Reply:
If rowhammer works all bets are off, your hardware has been broken and nothing is known any more. It is outside the scope of the architecture. Otherwise you have to add specific hardware on every single CHERI implementation to deal with cases that are architecturally impossible, which adds area. If you are an implementation that cares about memory corruption, use ECC.

Reply:
I agree with rowhammer being out of scope. I still think it's useful to define a specific outcome for all of the 129/65 input bits that are used by the instruction even if those inputs can never occur in a correct execution. Alternatively, we could change this to explicitly say the outcome is undefined. However, I think for cases that don't add much/any HW complexity having a well defined outcome is nicer.

Reply:
yes - this was the main purpose - so that we can 100% specify what happens for all inputs - just like in [the other PR](https://github.com/riscv/riscv-cheri/pull/285) - it's extremely useful for verification.

Reply:
merging as it's in line with the previous PR and is very useful for verification